src/main/java/io/jenkins/plugins/orka/helpers/FormValidator.java

```diff
@@ -90,15 +91,33 @@ public FormValidation doCheckNode(String node, String orkaEndpoint, String orkaC
         return FormValidation.error("There are no available nodes");
     }
 
-    public FormValidation doTestConnection(String credentialsId, String endpoint, boolean useJenkinsProxySettings, 
+    public FormValidation doCheckMemory(String memory) {
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+
+        try {
+            if (StringUtils.isBlank(memory) || StringUtils.equals(memory, "auto")) {
+                return FormValidation.ok();
+            }
+            if (Integer.parseInt(memory) < 1) {
+                return FormValidation.error("Memory should be greater than 0");
+            }
+            return FormValidation.ok();
+        } catch (Exception e) {
+            logger.log(Level.WARNING, "Exception in doCheckMemory", e);
+        }
+
+        return FormValidation.error("Memory should be greater than 0");
```

Comment:
Also, now that "auto" is default.
Why not change the msg to `Memory should be "auto" or greater than 0`

Reply:
And seems we only allow ints.
So it should be "auto" or integer greater than 0.

<img width="333" alt="image" src="https://user-images.githubusercontent.com/2429336/166896761-7b2c31ac-bbea-42cb-a5d8-f182adceaab6.png">

Reply:
That being said, should we allow integers only? Aren't we allowing any numbers in the CLI?